content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Customise the Helm Installation
+description: Customise the Istio installation with the included Helm chart.
```

Comment:
Custonise -> Customize

Probably better as:

"Describes the options available when installing Istio using the included Helm chart.